### PR TITLE
[BPK-3051] Add typography page

### PR DESCRIPTION
**Pull request by georgegillams**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] [Links platform metadata](https://github.com/Skyscanner/backpack-docs/blob/master/docs/src/layouts/links.js)

Screenshot_2019-11-04 Typography Backpack

### CODE REVIEW

File: docs/src/pages/TypographyPage/hello-relative.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  width: calc(100% + 2 * #{$bpk-spacing-lg});
+  overflow: hidden;
+  flex-direction: column;
+
+  @include bpk-breakpoint-mobile {
+    width: calc(100% + #{$bpk-spacing-lg});
+  }
+
+  &__text {
+    width: max-content;
```
Comment (georgegillams) [inline docs/src/pages/TypographyPage/hello-relative.scss]: This doesn't work on IE. Maybe we should include an IE fallback?

File: docs/src/pages/TypographyPage/hello-relative.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    font-size: 2 * $bpk-spacing-xxl;
+    line-height: 2 * $bpk-spacing-xxl;
+
+    &--medium {
+      font-weight: 500;
```
Comment (georgegillams) [inline docs/src/pages/TypographyPage/hello-relative.scss]:
```suggestion
      font-weight: $bpk-font-weight-bold;
```

File: docs/src/pages/TypographyPage/type-scale.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+.bpk-docs-type-scale {
+  display: flex;
+  width: calc(100% + 2 * #{$bpk-spacing-lg});
+  margin-bottom: -2 * $bpk-spacing-lg !important;
```
Comment (georgegillams) [inline docs/src/pages/TypographyPage/type-scale.scss]: This should be applied by the page css, not the component

File: docs/src/pages/TypographyPage/type-scale.scss [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  overflow: hidden;
+
+  @include bpk-breakpoint-mobile {
+    width: calc(100% + #{$bpk-spacing-lg});
+    margin-bottom: -#{$bpk-spacing-lg} !important;
```
Comment (georgegillams) [inline docs/src/pages/TypographyPage/type-scale.scss]: same here

File: docs/src/pages/TypographyPage/type-scale.scss [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+      color: $bpk-color-abisko;
+    }
+
+    &--annotation {
+      color: $bpk-color-sky-gray-tint-03;
+    }
+  }
+
+  &__table {
+    width: 200rem;
```
Comment (georgegillams) [inline docs/src/pages/TypographyPage/type-scale.scss]: Add comments to explain what these are for!

File: docs/src/pages/TypographyPage/typography-page.scss [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+.bpk-docs-typography-page {
+  display: flex;
+  margin-left: -$bpk-spacing-sm;
```
Comment (georgegillams) [inline docs/src/pages/TypographyPage/typography-page.scss]: This should be removed, unless it's here to make it consistent with other guideline pages.

**Comment by tiagoengel** [comment]
Looking good, can we also copy the font definition from `index.hml` to `template.js` so relative will be enabled in prod as well? It will be a bit odd to have this page in prod using the old font :D

File: docs/src/routes/Routes.js [hunk 7]
```diff
@@ -162,6 +163,7 @@ export const ROUTES_MAPPINGS = [
       { path: ROUTES.PHOTOGRAPHY, component: PhotographyPage },
       { path: ROUTES.TONE_OF_VOICE, component: ToneOfVoicePage },
       { path: ROUTES.STRATEGY, component: StrategyPage },
+      { path: ROUTES.TYPOGRAPHY, component: TypographyPage },
```
Comment (ojcurtis) [inline docs/src/routes/Routes.js]: Being picky here - Could this be put in the correct order?

File: docs/src/pages/TypographyPage/HelloRelative.js [hunk 8]
```diff
@@ -0,0 +1,84 @@
+/*
```
Comment (ojcurtis) [inline docs/src/pages/TypographyPage/HelloRelative.js]: Being a pain - could you rename this file to `FontFace` to make it more descriptive?

File: docs/src/pages/TypographyPage/HelloRelative.js [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import BpkText from 'bpk-component-text';
+import { cssModules } from 'bpk-react-utils';
+
+import STYLES from './hello-relative.scss';
```
Comment (ojcurtis) [inline docs/src/pages/TypographyPage/HelloRelative.js]: Being a pain - could you rename this file to `font-face` to make it more descriptive?

**Pull request merged**